sessions/react-effects-and-fetch/iss-tracker/README.md

```diff
@@ -0,0 +1,38 @@
+# React Effects and Fetch: ISS Tracker
+
+In this challenge we will build a small app which displays the current location of the international space station. We will also practice our `useEffect` skills.
+
+## Task
+
+We already have a state called `coords` in the `App` component which will hold the longitude and latitude values of the ISS position.
+
+1. Write the function `getISSCoords`. Fetch from the given url saved in the constant `URL`. After fetching successfully update the `coords` state accordingly.
+
+2. The refresh button already executes `getISSCoords` when pressed. Test if the function works as expected. (HINT: You can check if the position is displayed correctly by double checking on [this website](http://open-notify.org/Open-Notify-API/)).
+
+3. With the help of `useEffect` set an interval (with `setInterval`) to update the ISS coords every 5 seconds. The interval should be installed only on the first render of the app.
+   (HINT: Research `setInterval` on MDN if you don't know it)
+
+4. The `setInterval` function returns an ID of the respective interval. You can clear the interval by calling `clearInterval` with this id. Inside the useEffect, return a cleanup function, which executes this `clearInterval` function with the correct ID.
```

Comment:
I don't understand why we are doing this here: I see that the cleanup function is explained in the handout, but why is it useful in this specific example?

As far as I can see, the cleanup function removes the timer, so the data are not fetched every 5 seconds anymore. Why do we break the functionality we've just implemented? Only to train how to write a cleanup function?

If so, the question will raise what the purpose of this cleanup is. I think we should add an explanation here.

Reply:
I wrote a short explanation in the exercise description and additional not in the handout about the cleanup function